src/chrome/content/monitor.js

```diff
@@ -252,10 +252,33 @@ Monitor.prototype.isHTML = function (doc) {
     is_html = (contentType == 'text/html' ||
                contentType == 'text/xhtml' ||
                contentType == 'application/xhtml+xml');
+
+    var blacklisted_domains = itsalltext.preferences.blacklist_domains.split("\n");
+    var pattern,
+        domain_is_blacklisted = false;
```

Comment:
Can you make this a separate method?  And it should probably be called something like `isDomainBlacklisted()` take a domain as a string.

In theory, I should be writing tests for this stuff, but it wasn't really a thing when I started this project.